.azure/modules/containerAppEnv/main.bicep

```
@@ -22,6 +22,16 @@ param userAssignedIdentityId string
 @description('Whether zone redundancy should be enabled for the container app environment')
 param zoneRedundancyEnabled bool
 
+@description('Workload profiles to enable in the container app environment')
+param workloadProfiles array = [
+  {
+    name: 'Consumption'
+    workloadProfileType: 'Consumption'
+    minimumCount: 0
```

```suggestion
    minimumCount: 1
```

Bør det være minst 1 for disse miljøa? 👀 
Det blir det samme som replica-count?